contents/customers/mintlify.md

```diff
@@ -0,0 +1,59 @@
+---
+title: How Mintlify launched user-facing analytics, powered by PostHog
+customer: Mintlify
+logo: ../images/customers/mintlify/logo.png
+featuredImage: ../images/customers/mintlify/featured.png
+industries:
+    - SaaS, DevTool
+users:
+    - Leadership
+    - Engineering
+    - Product
+    - Users
+toolsUsed:
+    - Session Recording
+    - Autocapture
+    - PostHog Cloud
+    - API
+---
+
+Mintlify is a docs startup used by teams at [Y Combinator](/customers/ycombinator), Explo and  MindsDB to power their product and technical documentation. It helps teams with rapidly evolving products to create rich, well structured docs which convert users and empower colleagues. 
+
+Mintlify has used PostHog internally since launch, making regular use of both the product analytics and session recording tools to help it build better products. 
+
+“We use the product analytics tools, such as funnels, a lot,” says Han Wang, Mintlify’s CEO and Founder. “We also use session recordings too. I actually had to use it last night, to investigate errors on a particular setup. We’ve found recordings particularly useful for support issues like that, but we’re constantly tracking things… Is this CTA engaging? How popular is this page? We monitor everything in PostHog to help us decide where to focus.”
+
+<BorderWrapper>
+    <Quote
+        imageSource="/images/customers/han.png"
+        size="md"
+        name="Han Wang"
+        title="Founder & CEO, Mintlify"
+        quote={`“You can quote me on this: PostHog is awesome. It’s a great tool. I’ve used a bunch of different analytics platforms in the past and PostHog stands out for it’s developer friendliness and user experience. I really, really love it.”`}
+    />
+</BorderWrapper>
+
+## Session recording, analytics and user feedback
+
+After seeing first-hand how analytics could help them build better products, Mintlify decided to explore ways to give users this same ability within their own product. Rather than building the feature from scratch, the team identified a simpler solution: use PostHog. 
```

Comment:
```suggestion
After seeing first-hand how analytics could help them build better products, Mintlify decided to explore ways to provide users this same ability within their own product. Rather than building the feature from scratch, the team identified a simpler solution: use PostHog. 
```